### ISSUE TITLE
Update SA1137 to check braces of object and collection initializers

### ISSUE THREAD

**Issue by LiangZugeng** [DotNetAnalyzers/StyleCopAnalyzers#2747]
In the image below, the curly brackets doesn't align properly in a multiple-line statement, but StyleCop didn't catch it, is it intentionally ignored or there is no rule yet to check such kind of style issue.

![image](https://user-images.githubusercontent.com/20870130/43336464-949a8c6a-9203-11e8-811a-54c9a64f6f2e.png)

```
namespace StyleCopTest
{
    using System;

    /// <summary>
    /// Program.
    /// </summary>
    public static class Program
    {
        /// <summary>
        /// Main function.
        /// </summary>
        public static void Main()
        {
            Console.WriteLine("Hello World!");

            Student s = new Student
            {
                Name = "James",
           };
        }
    }
}
```

**Comment by sharwell** [comment]
This is a bug in SA1137, very closely related to #2665.

**Comment by LiangZugeng** [comment]
Thanks for your prompt response @sharwell , I will wait for the bug fix.
